src/server/api/user.js

```diff
@@ -165,7 +167,8 @@ export const resolvers = {
       "isSuspended"
     ]),
     isSuperadmin: (userRecord, _, { user: authUser }) => {
-      if (userRecord.id !== authUser.id) throw new ForbiddenError();
+      if (userRecord.id !== authUser.id && !authUser.is_superadmin)
```

Comment:
Question: See previous comment. Possibly may want users with lower permissions to have access to who's a superadmin or not.